test/index.js

```diff
@@ -0,0 +1,57 @@
+const proxyquire = require('proxyquire').noCallThru()
+const assert = require('assert')
+
+
+describe('main', () => {
+  describe('getStreams returns data from AWS', () => {
+    let main
+
+    before(() => {
```

Comment:
Dang. I tried that 17 days ago and it didn't work https://github.com/crccheck/kinesis-console-consumer/issues/4#issuecomment-250058937

But I know a little more now. I'll try again :(
I'll just have to create a new instance of `AWS.Kinesis` every function call I guess. But I **really** don't want to make my code harder to read just to accommodate some dude's test library.